6B/script.js

```diff
@@ -0,0 +1,116 @@
+/*
+    Tucker Kent
+    script.js
+    19SP_INFO_2134_WW Online - JavaScript II
+    Thoendel
+    15 April 2020 -- updated from 8 April 2020
+*/
+
+window.addEventListener("load", (event) => { //window event listener added upon load event
+
+    const input = document.getElementById("userInput"); //set input element to constant variable s
+    const convertLink = document.getElementById("action"); //set anchor element to constant variable 
+    const errorHolder = document.getElementById("errorHolder"); //set div element to constant variable 
+    const errorList = document.getElementById("errorList"); //set UL element to constant variable 
+    const convertFromHolder = document.getElementById("convertFromHolder"); //set div element to constant variable 
+    const convertToHolder = document.getElementById("convertToHolder"); //set div element to constant 
+    const convertFrom = document.getElementById("convertFrom"); //set select element to constant variable
+    const convertTo = document.getElementById("convertTo"); //set select element to constant variable 
+    const resultsHolder = document.getElementById("resultHolder"); //set div element to constant variable 
+
+    input.addEventListener("keyup", (e) => { //event listener added on keyup event to check input
+        errorList.innerHTML = ""; //resetting errorList to blank to erase previous errors
+        if(input.value === ""){ //if input is blank
+            displayError("You must enter a unit to convert"); //calling method to add an error message to the errorlist and make the errorHolder div visible
+            input.focus(); //setting focus to the input field
+        }
+        if(isNaN(input.value)){
+            displayError("You must enter a whole number to convert"); //calling method to add an error message to the errorlist and make the errorHolder div visible
```

Comment:
This error message is a bit misleading. You do accept decimal points - not just whole numbers.